main.py

```diff
@@ -85,8 +94,145 @@ def recording_file(rec_file):
     run = subparsers.add_parser("run", description="Run the model on the given data")
     run.add_argument("saved_model", nargs=1, type=existing_model)
     run.add_argument("--output-file", "-o", nargs="?", default=None, type=str)
-    run.add_argument("recordings", nargs="+", type=recording_file)
+    run.add_argument("recording", nargs="+", type=recording_file)
```

Comment:
What's the recording arg for?

Reply:
The recording is the recording you're evaluating on (i.e. the .wav file to classify)